spec/db/models.rb

```diff
@@ -27,24 +27,45 @@ class Author < VitualTotalTestBase
   virtual_total :total_recently_published_books, :recently_published_books
   virtual_aggregate :sum_recently_published_books_rating, :recently_published_books, :sum, :rating
 
-  # virtual_total using a virtual_has_many
+  # virtual_total using a virtual_has_many with an array of associations
+  # LAZY: sorry for lack of creativity, this could have been done with a where
```

Comment:
I wouldn't call this "LAZY", but I would say it is a use case of a poorly optimized `virtual_attribute`.

This comment is also redundant (regarding "creativity") when paired with the one inside the method.

* * *

Idea in the future to replace this would be to do something like `book_type`, where "valid types" are an array in ruby, and they are formatted via `formatted_book_type`, and "Unknown" types that aren't in the array are just formatted to "Other".

But I think that idea deserves another PR.